File: AD-HOC/1217/main.py
total_price, total_weight = 0.0, 0.0
n = int(input())
for day in range(1, n + 1):
    total_price += float(input())
    weight = len(input().split())
    print(f'day {day}: {weight} kg')
    total_weight += weight
print(f'{total_weight / n:.2f} kg by day')
print(f'R$ {total_price / n:.2f} by day')

File: AD-HOC/1217/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double totalPrice = 0.0;
        double totalWeight = 0.0;
        int n = scanner.nextInt();
        for (int day = 1; day < n + 1; day++) {
            totalPrice += scanner.nextDouble();
            scanner.nextLine();
            int weight = scanner.nextLine().trim().split(" ").length;
            System.out.printf("day %d: %d kg\n", day, weight);
            totalWeight += weight;
        }
        System.out.printf("%.2f kg by day\n", totalWeight / n);
        System.out.printf("R$ %.2f by day\n", totalPrice / n);
        scanner.close();
    }
}
